package.json

```diff
@@ -128,21 +130,30 @@
     "url": "https://github.com/bahmutov/cypress-react-unit-test.git"
   },
   "dependencies": {
+    "@babel/core": "7.4.5",
```

Comment:
why do we include babel and babel-loader as prod dependencies? I assume the React project has them.